src/app/component/ProductReviews/ProductReviews.style.scss

```diff
@@ -68,6 +68,7 @@
         top: 0;
         bottom: 0;
         height: max-content;
+        max-height: 4.1rem;
```

Comment:
1. Why 4.1rem? Other buttons have bigger height. Limiting the height causes that the text doesn't have a propper vertical alignment.
![image](https://user-images.githubusercontent.com/73945186/101900367-06db7900-3bb8-11eb-901d-1cba8f84582f.png)

2. If both top and bottom css properties are set, the element height starts stretching.